File: linear_search.py
# linear search algorithm
array =[2,4,5,6,3,7,8,20]
seach_value = 5
def Linear_Search(arr,value):
    for i in range(len(arr)):
        if(arr[i]== value):
            return i
    return -1

print(Linear_Search(array,seach_value)   )            

File: linear_search.js
// linear_search algorithm 
var list =[2,4,5,7,8,9,10,64]
var value =11
function linearSearch(list,value){
    for(var i= 0;i<list.length;i++){
        if(list[i]==value){
            return i
        }
    }
    return -1
}
console.log(linearSearch(list,value))